ui/src/app/pages/admin/workspace/basic-information.spec.tsx

```diff
@@ -0,0 +1,108 @@
+import '@testing-library/jest-dom';
+
+import * as React from 'react';
+
+import {
+  FeaturedWorkspaceCategory,
+  WorkspaceActiveStatus,
+  WorkspaceAdminApi,
+} from 'generated/fetch';
+
+import { screen } from '@testing-library/react';
+import userEvent, { UserEvent } from '@testing-library/user-event';
+import {
+  registerApiClient,
+  workspaceAdminApi,
+} from 'app/services/swagger-fetch-clients';
+import { serverConfigStore } from 'app/utils/stores';
+
+import defaultServerConfig from 'testing/default-server-config';
+import {
+  expectButtonElementDisabled,
+  expectButtonElementEnabled,
+  renderWithRouter,
+} from 'testing/react-test-helpers';
+import { WorkspaceAdminApiStub } from 'testing/stubs/workspace-admin-api-stub';
+import { workspaceStubs } from 'testing/stubs/workspaces';
+
+import { BasicInformation } from './basic-information';
+
+describe('BasicInformation', () => {
+  const activeStatus = WorkspaceActiveStatus.ACTIVE;
+  let user: UserEvent;
+  let workspace;
+  const component = () => {
+    return renderWithRouter(
+      <BasicInformation
+        {...{ workspace, activeStatus }}
+        reload={async () => Promise.resolve()}
+      />
+    );
+  };
+
+  beforeEach(() => {
+    workspace = workspaceStubs[0];
+    user = userEvent.setup();
+    serverConfigStore.set({
+      config: {
+        ...defaultServerConfig,
+      },
+    });
+    registerApiClient(WorkspaceAdminApi, new WorkspaceAdminApiStub());
+  });
+
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it('renders', async () => {
+    component();
+    expect(
+      await screen.findByRole('heading', { name: /basic information/i })
+    ).toBeInTheDocument();
+  });
+
+  it('should show unpublished workspace', async () => {
+    component();
+
+    expect(screen.getByText('Select a category...')).toBeInTheDocument();
+    expectButtonElementDisabled(
+      screen.getByRole('button', { name: 'Publish' })
+    );
+    expectButtonElementDisabled(
+      screen.getByRole('button', { name: /unpublish/i })
+    );
+  });
+  it('should show published workspace', async () => {
+    workspace.featuredCategory = FeaturedWorkspaceCategory.COMMUNITY;
+    component();
+    expect(await screen.findByText('Community')).toBeInTheDocument();
```

Comment:
Should we use the react test helper getDropdownOption or something to ensure Community is from dropdown

Reply:
Since this is an exact match and there are not any other known instances of 'Community' in the component, I think that this is a safe approach for now. Happy to talk it through.